server/app_sudoku/domain/sudoku_utils.py

```diff
@@ -171,7 +178,28 @@ def solve_sudoku(board: List[int]) -> List[int]:
     output = search(output)
     if not output:
         return []
-    return sudoku_to_list(output)
+    output_list = sudoku_to_list(output)
+    return output_list if verify_solution(output_list) else []
+
+
+def has_unique_solution(board: List[int]) -> bool:
+    output = parse_board(board)
+    if not output:
+        return False
```

Comment:
Här kan du se hur Codecov verktyget kan vara till nytta. Det indikerar att du missat att testa ett `if` statement i koden. Så vi har inga automatiserade test som verkligen verifierar att `if not output` kommer att returnera `False`.

Självkart ska man inte blint följa verktyget utan bedömma från fall till fall om det är relevant att lägga till test. Och den bedömningen lämnar jag över åt dig. 😉